src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -64,140 +65,142 @@ const defaultProps = {
     reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
 };
 
-class WorkspaceReimburseView extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            currentRatePerUnit: this.getCurrentRatePerUnitLabel(),
-        };
-    }
-
-    componentDidMount() {
-        this.fetchData();
-    }
-
-    componentDidUpdate(prevProps) {
-        if (prevProps.policy.customUnits !== this.props.policy.customUnits || prevProps.preferredLocale !== this.props.preferredLocale) {
-            this.setState({currentRatePerUnit: this.getCurrentRatePerUnitLabel()});
-        }
-
-        const reconnecting = prevProps.network.isOffline && !this.props.network.isOffline;
-        if (!reconnecting) {
-            return;
-        }
+function WorkspaceReimburseView(props) {
+    const [currentRatePerUnit, setCurrentRatePerUnit] = useState('');
+    const prevIsOffline = usePrevious(props.network.isOffline);
+    const viewAllReceiptsUrl = `expenses?policyIDList=${props.policy.id}&billableReimbursable=reimbursable&submitterEmail=%2B%2B`;
+    const distanceCustomUnit = _.find(lodashGet(props.policy, 'customUnits', {}), (unit) => unit.name === 'Distance');
+    const distanceCustomRate = _.find(lodashGet(props.distanceCustomUnit, 'rates', {}), (rate) => rate.name === 'Default Rate');
+
+    const getNumericValue = useCallback(
+        (value) => {
+            const propsToLocaleDigit = props.toLocaleDigit;
+            const numValue = parseFloat(value.toString().replace(propsToLocaleDigit('.'), '.'));
+            if (Number.isNaN(numValue)) {
+                return NaN;
+            }
+            return numValue.toFixed(3);
+        },
+        [props.toLocaleDigit],
+    );
+
+    const getRateDisplayValue = useCallback(
+        (value) => {
+            const propsToLocaleDigit = props.toLocaleDigit;
```

Comment:
Same as above

Reply:
Yes we need to destructure the props else it will give eslint warning for exhaustive deps
You can see discussions [over here](https://github.com/facebook/react/issues/16265)